sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/LogsHelper.cs

```diff
@@ -80,6 +83,45 @@ internal static string GetMessageAndSetProperties(LogRecord logRecord, IDictiona
             return message;
         }
 
+        internal static void WriteScopeInformation(LogRecord logRecord, IDictionary<string, string> properties)
+        {
+            StringBuilder builder = null;
+            int originalFormatDepth = 1;
+            logRecord.ForEachScope(ProcessScope, properties);
+
+            void ProcessScope(LogRecordScope scope, IDictionary<string, string> properties)
+            {
+                if (scope.Scope is IEnumerable<KeyValuePair<string, object>> stateDictionary)
+                {
+                    foreach (KeyValuePair<string, object> scopeItem in scope)
+                    {
+                        if (scopeItem.Key == "{OriginalFormat}")
+                        {
+                            properties.Add($"OriginalFormatScope_{originalFormatDepth++}", Convert.ToString(scope.Scope.ToString(), CultureInfo.InvariantCulture));
```

Comment:
it looks like the depth is incremented *only* for OriginalFormat.. is that intentional?

Reply:
Yes. It is intentional. There could be multiple original format, this helps track all of it.

```
{
"name": "Message",
...
"data": {
	"baseType": "MessageData",
	"baseData": {
		"message": "3.1",
		"severityLevel": "Verbose",
		"properties": {
			"OriginalFormat": "3.1",
			"WithValue": "12345",
			"OriginalFormatScope_1": "Formatted 12345",
                        "SomeValue": "56789",
                         "OriginalFormatScope_2":"2nd Formatted 56789"
			"ViaDictionary": "100",
			"Scope": " => Some name => 42"
		},
		"ver": 2
	}
}
```

Code

```
using (_logger.BeginScope("Some name"))
{
    _logger.LogInformation("1");
    using (_logger.BeginScope(42))
    {
        _logger.LogInformation("2");
        using (_logger.BeginScope("Formatted {WithValue}", 12345))
        {
            using (_logger.BeginScope("2nd Formatted {SomeValue}", 56789))
            {
                _logger.LogInformation("3.1");
            }
        }
    }
}
```